server/service/src/programs/program_event.rs

```diff
@@ -90,10 +90,11 @@ fn remove_event_stack(
                 .active_end_datetime(DatetimeFilter::equal_to(datetime)),
         ),
         Some(ProgramEventSort {
-            key: ProgramEventSortField::ActiveEndDatetime,
+            key: ProgramEventSortField::ActiveStartDatetime,
             desc: Some(true),
```

Comment:
When filterning for a fixed end time it doesn't make sense to sort by the end time. It was then up to the mercy of the DB in which order events where returned.